integration-tests/chopsticks/tsconfig.json

```diff
@@ -0,0 +1,14 @@
+{
+  "compilerOptions": {
+    "target": "es2016",
+    "lib": ["es2019"],
+    "module": "NodeNext",
+    "rootDir": "src",
+    "outDir": "./bin",
```

Comment:
We don't need this, do we?

Reply:
Updated ts config in: [35f7268](https://github.com/KILTprotocol/kilt-node/pull/614/commits/35f7268a42ccdf11ce2809a8d2f3fccefc1bdb7b)